Basic/APIDiorama/README.md

```diff
@@ -0,0 +1,15 @@
+# API Diorama
```

Comment:
A minor and a personal preference but I'd suggest calling the project "Examples", "Demos" or "Showcase" instead of "Diorama" since they sound more common and obvious compared to "Diorama".

So, I'd like to propose "API Showcase" as my personal pick but let's have this discussion.

Reply:
I agree on a name change, however this PR is not the ideal place for it (this PR can't be merged if there are unresolved comments). I moved the discussion to Slack, [in the Samples' team channel](https://unity.slack.com/archives/G01H7JP4AP2/p1679500191079979) (unfortunately it's private), so they can pick a better name